examples/rl/sb3_simple.py

```diff
@@ -0,0 +1,179 @@
+""" Optuna example that optimizes the hyperparameters of
+a reinforcement learning agent using A2C implementation from Stable-Baselines3
+on a OpenAI Gym environment.
+
+This is a simplified version of what can be found in https://github.com/DLR-RM/rl-baselines3-zoo.
+
+You can run this example as follows:
+    $ python sb3_simple.py
+
+"""
+from typing import Any
+from typing import Dict
+
+import gym
+from stable_baselines3 import A2C
+from stable_baselines3.common.callbacks import EvalCallback
+import torch
+import torch.nn as nn
+
+import optuna
+from optuna.pruners import MedianPruner
+from optuna.samplers import TPESampler
+
+
+N_TRIALS = 100
+N_JOBS = 2
+N_STARTUP_TRIALS = 5
+N_EVALUATIONS = 2
+N_TIMESTEPS = int(2e4)
+EVAL_FREQ = int(N_TIMESTEPS / N_EVALUATIONS)
+N_EVAL_EPISODES = 3
+
+ENV_ID = "CartPole-v1"
+
+DEFAULT_HYPERPARAMS = {
+    "policy": "MlpPolicy",
+    "env": ENV_ID,
+}
+
+
+def sample_a2c_params(trial: optuna.Trial) -> Dict[str, Any]:
+    """Sampler for A2C hyperparameters."""
+    gamma = 1.0 - trial.suggest_float("gamma", 0.0001, 0.1, log=True)
+    max_grad_norm = trial.suggest_float("max_grad_norm", 0.3, 5.0, log=True)
+    gae_lambda = 1.0 - trial.suggest_float("gae_lambda", 0.001, 0.2, log=True)
+    n_steps = 2 ** trial.suggest_int("exponent_n_steps", 3, 10)
+    learning_rate = trial.suggest_loguniform("lr", 1e-5, 1)
+    ent_coef = trial.suggest_loguniform("ent_coef", 0.00000001, 0.1)
```

Comment:
These lines can be converted to use `suggest_float` as follows.
```suggestion
    learning_rate = trial.suggest_float("lr", 1e-5, 1, log=True)
    ent_coef = trial.suggest_float("ent_coef", 0.00000001, 0.1, log=True)
```

Reply:
is that the recommended way now?
I personally find it less clear that you have to notice `log=True` to know what is happening.

Reply:
Yes, it is. Currently, we recommends to use `suggest_float` rather than `suggest_uniform`, `suggest_loguniform, and `suggest_discrete`.